llama_index/finetuning/types.py

```diff
@@ -44,3 +44,15 @@ def get_finetuned_model(
         self, model_name: str, top_n: int = 3
     ) -> SentenceTransformerRerank:
         """Gets fine-tuned Cross-Encoder model as re-ranker."""
+
+
+class BaseCohereRerankerFinetuningEngine(ABC):
```

Comment:
we probably don't need this base class - we can just take the existing `BaseCrossEncoderFinetuningEngine`, and have the output type of `get_finetuned_model` be `BaseNodePostprocessor`

Reply:
we should figure out the right base abstractions in the next PR